apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Trace/CloudTraceExtensionTest.cs

```diff
@@ -13,52 +13,20 @@
 // limitations under the License.
 
 using Google.Cloud.Diagnostics.Common;
-using Google.Cloud.Diagnostics.Common.Tests;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Internal;
 using Moq;
 using System;
 using Xunit;
 
-using ProjectId = Google.Cloud.Diagnostics.AspNetCore.CloudTraceExtension.ProjectId;
-using TraceProto = Google.Cloud.Trace.V1.Trace;
-
-namespace Google.Cloud.Diagnostics.AspNetCore.Tests.Trace
+namespace Google.Cloud.Diagnostics.AspNetCore.Tests
 {
     public class CloudTraceExtensionTest
     {
-        private const string ProjectId = "pid";
-        private const string TraceId = "105445aa7843bc8bf206b12000100f00";
-        private const ulong SpanId = 81237123;
-
-        /// <summary>
-        /// Creates an <see cref="IServiceProvider"/> with all the needed dependencies to
-        /// create an <see cref="IManagedTracer"/>.
-        /// </summary>
-        /// <param name="context">The trace header context.</param>
-        /// <param name="rateLimiterShouldTrace">True if the rate limiter should allow tracing.</param>
-        private Mock<IServiceProvider> CreateProviderForCreateManagedTracer(
-            TraceHeaderContext context, bool rateLimiterShouldTrace)
-        {
-            Mock<IConsumer<TraceProto>> mockConsumer = new Mock<IConsumer<TraceProto>>();
-            Mock<IServiceProvider> mockProvider = new Mock<IServiceProvider>();
-
-            mockProvider.Setup(p => p.GetService(typeof(TraceHeaderContext))).Returns(context);
-            mockProvider.Setup(p => p.GetService(typeof(ProjectId))).Returns(new ProjectId { Id = ProjectId });
-            mockProvider.Setup(p => p.GetService(typeof(TraceIdFactory))).Returns(TraceIdFactory.Create());
-            mockProvider.Setup(p => p.GetService(typeof(IConsumer<TraceProto>))).Returns(mockConsumer.Object);
-
-            var rateLimiter = TraceUtils.GetRateLimiter(rateLimiterShouldTrace ? 1001 : 999);
-            var rateLimitingFactory = new RateLimitingTraceOptionsFactory(rateLimiter);
-            mockProvider.Setup(p => p.GetService(typeof(RateLimitingTraceOptionsFactory)))
-                .Returns(rateLimitingFactory);
-            return mockProvider;
-        }
-
         [Fact]
         public void CreateTraceHeaderContext()
         {
-            string header = $"{TraceId}/{SpanId};o=1";
+            string header = $"105445aa7843bc8bf206b12000100f00/81237123;o=1";
```

Comment:
I would probably leave the Trace/SpanId constants and not hardcode them here, just in case other tests are added in the future we won't duplicate the values.